src/components/dashboard/ServiceList.js

```diff
@@ -1,18 +1,17 @@
-import React from 'react';
+import React, {Component} from 'react';
```

Comment:
Isn't this ServiceList a stateless component. I think this {Component} import is not necessary.